12-watchtower-API.md

```diff
@@ -127,122 +129,167 @@ This message contains all the information regarding the appointment that the cli
-* upon receiving an `end_block` too far away in the future:
-	* SHOULD reject the appointment.
-* upon receiving a `dispute_delta` too small:
-	* SHOULD reject the appointment.
-* upon receiving an `encrypted_blob` of non-feasible size:
-	* MUST reject the appointment.
-* upon receiving a `cipher` that he does not implement:
-	* MUST reject the appointment. 
-* upon receiving a `transaction_size` too big:
-	* MUST reject the appointment.
-* upon receiving a `transaction_size` too small:
-	* MUST reject the appointment.
-* upon receiving a `transaction_fee` too low:
-	* SHOULD reject the appointment.
+The receiving node MUST reject the appointment if:
 
-* if `qos` was agreed on `wt_init`:
-	* if `op_customer_signature_algorithm` is missing:
-		* MUST reject the appointment.
-	* if `op_customer_signature_algorithm` does not match with one of the supported signing algorithms:
-		* MUST reject the appointment.
-	* if `op_customer_signature` is missing:
-		* MUST reject the appointment.
-	* if `op_customer_public_key` is missing:
-		* MUST reject the appopintment.
-	* if `op_customer_signature` cannot be verified using `op_customer_public_key`:
-		* MUST reject the appopintment.
-
-* if `qos` was not agreed on `wt_init`:
-	* if `op_customer_signature_algorithm` is present:
-		* SHOULD reject the appointment.
-	* if `op_customer_signature` is present:
-		* SHOULD reject the appointment.
-	* if `op_customer_public_key` is present:
-		* SHOULD reject the appopintment.
+* `locator` is not a `16-byte` value.
+* `start_block` is not an integer.
+* `start_block` is behind the current chain tip.
+* `end_block` is not an integer.
+* `end_block` is behind the current chain tip.
+* `dispute_delta` is not an integer.
+* `encrypted_blob` has non-feasible size.
+* `cipher` is not among the one he implements.
+* `transaction_size` is non-feasible.
 
+The receiving node SHOULD reject the appointment if:
+
+* `start_block` is too close to the current chain tip.
+* `start_block` is too far away in the future.
+* `end_block` is too far away in the future.
+* `dispute_delta` is too small.
+* `fee_rate` is too low.
+
+* if `qos` was agreed on `wt_init`:
+	The receiving node MUST also reject the appointment if:
+	* `op_customer_signature_algorithm` is missing.
+	* `op_customer_signature_algorithm` does not match with one of the supported signing algorithms.
+	* `op_customer_signature` is missing.
+	* `op_customer_public_key` is missing.
+	* `op_customer_signature` cannot be verified using `op_customer_public_key`.
+
+* if `qos` was NOT agreed on `wt_init`:
+	* The receiving node SHOULD also reject the appointment if:
+	* if `op_customer_signature_algorithm` is present.
+	* if `op_customer_signature` is present.
+	* if `op_customer_public_key` is present.
+
+The receiving node MAY accept the appointment otherwise.
+
+#### Range of values 
+
+- `start_block`: Absolute Block number 
+- `end_block`: Absolute Block number
+- `dispute_delta`: Relative Block number
```

Comment:
are we keeping this?

Reply:
This was part of your writing. I'm happy to take it out, I feel it has already been covered.

Reply:
could just remove it

Reply:
Ok